pysph/tools/ipy_viewer.py

```diff
@@ -156,12 +157,19 @@ def __init__(self, particlearray, particles_type):
             layout=widgets.Layout(width='240px', display='flex')
         )
         self.scalar.owner = particles_type
-        self.legend = widgets.ToggleButton(
+        self.scalar_cmap = widgets.Dropdown(
+            options=map(str, plt.colormaps()),
```

Comment:
Does this work on Python3?  Does options require a list or is any iterable good enough?